src/main/resources/public/js/components/store-install-app.js

```diff
@@ -46,7 +46,7 @@ class AppModal extends React.Component {
             buying: false,
             installing: false,
             isInstalled: false,
-            error: false
+            error: {status : false, http_status : 200}
```

Comment:
couldn't the return code be only managed via the HTTP response code returned from the backend ?